packages/parser/src/language/common/tokenBuilder.ts

```diff
@@ -3,7 +3,14 @@ import { DefaultTokenBuilder } from 'langium';
 
 import type { TokenType } from '../chevrotainWrapper.js';
 
-export class InfoTokenBuilder extends DefaultTokenBuilder {
+export class CommonTokenBuilder extends DefaultTokenBuilder {
```

Comment:
This should be `abstract class MermaidTokenBuilder`, as it's always extended in diagrams. 

The idea is 

`Mermaid...` -> Abstract classes that needs to be implemented
`Common...` -> Common implementation of a `Mermaid...` class, that can be used if the diagram does not need to extend.